CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ All notable changes to this project will be documented in this file.
 ### Quality
 * Improve code quality.
 * Extract STOMP client configuration to its own bean.
+* Refactor STOMP client service to simplify its implementation.
```

Comment:
To be moved to NEXT version ?

Reply:
Good catch!

Reply:
Good catch, here it is moved: https://github.com/iExecBlockchainComputing/iexec-worker/pull/492/commits/826880c14d6ea2ef63d94aa1d38d4a34b0aa8e5f.